eng/devices/uitests-shared.cake

```diff
@@ -0,0 +1,13 @@
+#load "../cake/helpers.cake"
+
+if (!IsCIBuild() && GetBuildVariable("workloads", "notset") == "notset")
```

Comment:
For UI tests lets just default to global workloads for now. I think we should just make this change everywhere. This isn't really a scenario that most users care about anymore.